prefs.js

```diff
@@ -29,7 +29,7 @@ class Settings {
             title: _("History Size"),
             adjustment: new Gtk.Adjustment({
                 lower: 1,
-                upper: 200,
+                upper: 1000,
```

Comment:
@Tudmotu in https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/issues/495 said he will accept raising the limit to 10k not 1k